climada/entity/exposures/crop_production.py

```diff
@@ -105,42 +92,45 @@
 FAO_FILE = "FAOSTAT_data_producer_prices.csv"
 FAO_FILE2 = "FAOSTAT_data_production_quantity.csv"
 
+YEARS_FAO = np.array([2000, 2018])
+"""Default years from FAO used"""
+
 # default output directory: climada_python/data/ISIMIP_crop/Output/Exposure
 # by default the hist_mean files created by climada_python/hazard/crop_potential are saved in
 # climada_python/data/ISIMIP_crop/Output/hist_mean/
 HIST_MEAN_PATH = os.path.join(DATA_DIR, 'ISIMIP_crop', 'Output', 'Hist_mean')
 OUTPUT_DIR = os.path.join(DATA_DIR, 'ISIMIP_crop', 'Output')
 
 
-class CropyieldIsimip(Exposures):
+class CropProduction(Exposures):
     """Defines agriculture exposures from ISIMIP input data and
     FAO crop price data"""
 
     _metadata = Exposures._metadata + ['crop']
 
     @property
     def _constructor(self):
-        return CropyieldIsimip
+        return CropProduction
 
     def set_from_single_run(self, input_dir=INPUT_DIR, filename=None, hist_mean=HIST_MEAN_PATH,
-                            bbox=BBOX, yearrange=(YEARCHUNKS[SCENARIO[1]])['yearrange'],
-                            cl_model=None, scenario=SCENARIO[1], crop=CROP[0], irr=IRR[0],
+                            bbox=BBOX, yearrange=(YEARCHUNKS['histsoc'])['yearrange'],
+                            cl_model=None, scenario='histsoc', crop=None, irr=None,
                             unit='USD', fn_str_var=FN_STR_VAR):
 
         """Wrapper to fill exposure from nc_dis file from ISIMIP
         Parameters:
             input_dir (string): path to input data directory
-            filename (string): name of the file to use
-            historic mean (array): historic mean yield per centroid
+            filename (string): name of the landuse data file to use,
```

Comment:
General comment for the whole file. There seems to be quite a few methods that rely on a specific pattern for the filenames. This would imply that the current module only works for a very specific set of data files. This is suboptimal if in the future the module should be extended to use other data sources. It can work if there is a clear documentation on how to encode the data and how to name the files. Otherwise, the module is fixed for this one set of data files and should be documented accordingly.